api/src/pcapi/alembic/versions/20230327T124205_3acdce9ec195_drop_procuct_cgu_not_null.py

```diff
@@ -0,0 +1,19 @@
+"""drop_procuct_cgu_not_null
```

Comment:
(détails) Pour un plus joli affichage dans `alembic history` : 

```suggestion
"""Drop duplicate constraint check_iscgucompatible_is_not_null on product
```